src/main/java/com/dev/sunny/model/Author.java

```diff
@@ -0,0 +1,26 @@
+package com.dev.sunny.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.List;
+
+@Getter
+@Setter
+@ToString
+@Entity
+public class Author {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    Long id;
+    String name;
+    String email;
+    List<String> publications;
+
```

Comment:
Add constructor with all attribute

 - Should be without id attribute.
 - Use @NoArgsConstructor to provide default constructor.

Reply:
Resolved above changes